airflow/secrets/local_filesystem.py

```diff
@@ -74,7 +74,7 @@ def _parse_env_file(file_path: str) -> Tuple[Dict[str, List[str]], List[FileSynt
             # Ignore comments
             continue
 
-        var_parts: List[str] = line.split("=", 2)
+        var_parts: List[str] = line.split("=", 1)
```

Comment:
It would be easier to use `partition` here.

```python
key, sep, value = line.partition("=")
if not sep:
    errors.append(...)
    continue
if not key:
    errors.append(...)
```

Reply:
Thanks for your comment, it did looks more elegant, let me update the code by using partition.